layouts/default.vue

```diff
@@ -58,7 +58,7 @@
       </v-btn>
     </v-toolbar>
     <v-content>
-      <v-container>
+      <v-container fill-height>
```

Comment:
Add `class="pa-0 ma-0"` to the container to fill the page with just the map

Reply:
When I add class="pa-0 ma-0" the container collapses to 0 height. I've been using height=100% and width=100% for the map, but I think these take 100% of 0. I'm not sure how to get the map to fill the available area. I'm thinking with a flexbox and the grow property, but I haven't yet experimented with this.

Reply:
Scratch that, I was replacing fill-height. However, the v-container does not fill the width of the screen on desktop as it's the max-width property is set to 900px.